source/reference/minio-mc/mc-tag-set.rst

```diff
@@ -86,6 +86,21 @@ Parameters
 
       mc tag set myminio/mybucket/object.txt "key1=value1&key2=value2"
 
+.. mc-cmd:: --exclude-folders
+   :optional:
+
+   .. versionadded:: RELEASE.2024-01-11T05-49-32Z
+
+   When used with :mc-cmd:`~mc tag set --recursive`, causes :mc:`mc tag set` to *not* traverse child prefixes.
```

Comment:
I suggest making the **not** bold rather than just italics.

```suggestion
   When used with :mc-cmd:`~mc tag set --recursive`, causes :mc:`mc tag set` to **not** traverse child prefixes.
```